apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/create-entity-page.tsx

```diff
@@ -91,10 +111,23 @@ export const CreateEntityPage = ({ entityTypeId }: CreateEntityPageProps) => {
 
   const entityLabel = generateEntityLabel(draftEntitySubgraph);
 
+  const isQueryEntity = entityTypeId === QUERY_ENTITY_TYPE_ID;
+
+  const entityEditorProps: EntityEditorProps = {
+    setEntity: (entity) => {
+      updateEntitySubgraphStateByEntity(entity, setDraftEntitySubgraph);
+    },
+    draftLinksToCreate,
+    setDraftLinksToCreate,
+    draftLinksToArchive,
+    setDraftLinksToArchive,
+    entitySubgraph: draftEntitySubgraph,
+    readonly: false,
+    refetch: async () => {},
+  };
```

Comment:
I think you can revert this change, unless there is some benefit to keeping it

Reply:
Removed both here https://github.com/hashintel/hash/pull/2491/commits/5e1677d5e862f76ba67151b89393e840978a79a0